.github/workflows/release-pr.yml

```diff
@@ -0,0 +1,69 @@
+name: Create Release PR
+on:
+  create:
+    branches:
+    - 'release/v**'
+
+jobs:
+  build:
+    name: Create Release PR
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Calculate Tag
+      run: |
+        t=$(echo ${{ github.ref }} | sed -ne 's/.*\([0-9]\+\.[0-9]\+\.[0-9]\+\).*/\1/p')
+        if [ -z "$t" ]; then
+        echo "Malformed release branch name ${{ github.ref }}; failed to extract semVer tag"
+        return 1
+        fi
+        echo ::set-env name=TAG_NAME::"v${t}"
+
+    - name: Install release-notes
+      run: |
+        mkdir -p $HOME/deps
+        curl -Lo $HOME/release-notes.tar.gz https://github.com/gsquared94/release-notes/releases/download/v0.1.2/release-notes-linux-amd64.tar.gz
```

Comment:
I have a mild concern about requiring a fork.

Reply:
I forked that project since I wanted to be able to categorize the changelog and that feature was missing there. Couldn't follow the concern part. Should I have done this differently?